app/internal/page/choose_attorneys_address.go

```diff
@@ -75,6 +98,21 @@ func ChooseAttorneysAddress(logger Logger, tmpl template.Template, addressClient
 	}
 }
 
+func redirectPath(from string) string {
+	var path string
+
+	switch from {
+	case "summary":
+		path = chooseAttorneysSummaryPath
+	case "check":
+		path = checkYourLpaPath
+	default:
+		path = chooseAttorneysSummaryPath
+	}
+
+	return path
+}
```

Comment:
I'm guessing this pattern will appear in a few places that have a change link so may be worth extracting this to somewhere central for re-use once that happens.